app-frontend/src/app/components/map/labMap/labMap.html

```diff
@@ -49,8 +49,32 @@
         </div>
       </div>
     </div>
+    <div class="filler"></div>
+    <button class="btn btn-default btn-square btn-small"
+            title="Measure"
+            ng-click="$ctrl.toggleMeasurePicker($event)">
+      <span class="icon-check"></span>
+    </button>
+    <div class="measure-dropdown-control layer-picker"
+         ng-show="$ctrl.measurePickerOpen">
+      <div class="layer-picker-title">Pick Measurement</div>
+      <div class="dropdown-arrow"></div>
+      <div class="layer-picker-body">
+        <div class="layer-picker-option"
+             ng-click="$ctrl.toggleMeasure('Polygon')">
+          <div class="layer-picker-label">
+            Polygon
```

Comment:
I think it would be better to change the names of the options to their purpose:

 - Polyline => Distance
 - Polygon => Area

Reply:
@notthatbreezy done.